projects/vdk-plugins/vdk-impala/src/vdk/plugin/impala/templates/load/fact/insert/README.md

```diff
@@ -38,6 +38,10 @@ def run(job_input):
     # . . .
 ```
 
+The diagram below shows how the data from the source view is inserted into the target table, without checking for duplicate data.
+![Insert_template_diagram](https://user-images.githubusercontent.com/21333266/204575441-eafb10dc-d514-4f9d-9b74-763695c3afba.png)
```

Comment:
I'd really appreciate adding a link to the source as well (Something like `generated using this presentation` )

Reply:
Comment is added